### DISCUSSION TITLE
[spack] unable to disable sourceMaps?

### DISCUSSION THREAD

**Discussion by mrceperka** [swc-project/swc#4892] [Q&A] [answered]
Hello,

**Problem:** I am not able to disable `sourceMaps`, there are always generated.

**Setup:**
```js
// spack.config.js`
module.exports = config({
  // multiple entries
  entry,
  output: {
    path: __dirname + "/dist/swc"
  },
  options: {
    jsc: {
      target: "es5",
    },
    sourceMaps: false,
    inlineSourcesContent: false,
  },
});
```

**Question:** How can I disable `sourceMaps` if this is not a bug? :D

Thanks in advance :pray:

**Comment by gotham8x** [comment]
A half year and still no response. Webpack is still the king.

**Comment by mrceperka** [comment, marked as answer]
Probably resolved in https://github.com/swc-project/swc/issues/6519. Closing.

**Comment by electrovir** [comment]
This is not solved. #4562 references this problem but idk what happened to it because it was closed without much comment.
